### ISSUE TITLE
Place All Games stats at the top of Individuals report instead of bottom

### ISSUE THREAD

**Issue by ani-per** [ANadig/YellowFruit#67]
Right now, for tournaments with two stages (e.g. Round Robin and Playoffs), the Individual stats are displayed in chronological order with respect to stages (e.g. [here, Round Robin and then All Games](https://hsquizbowl.org/db/tournaments/9054/stats/all_games/individuals/)). It'd be more natural to place the All Games stats first, then have Round Robin and Playoffs stats underneath.

**Comment by jonahgreenthal** [comment]
I disagree. Individual awards are almost always based only on preliminary games, so that (or whatever phase(s) is (are) used for individual awards) should be the most prominent.

(I also think it's very weird for a single page to show the data from multiple phases this way, but that may simply be because I'm very used to seeing only one phase at a time.)

**Comment by ani-per** [comment]
> I disagree. Individual awards are almost always based only on preliminary games, so that (or whatever phase(s) is (are) used for individual awards) should be the most prominent.

You're right, I didn't consider that - my mistake. I was approaching from a mindset of one-off tournaments / side events, which are unlikely to have awards and where players are more concerned with their overall stats. Maybe this could be a configurable option with the "first-stage" scheme as default.

**Comment by ANadig** [comment]
Jonah identified my reasoning for doing it this way. Once you've done anything past prelim rounds -- playoffs, tiebreakers,  finals -- individual PPG totals aren't apples-to-apples, such that a full ranking of player PPGs is almost meaningless. (I think it's still worth putting in the report for completeness' sake.

Closing as not planned for now. I guess I could be open to introducing configurability, but I think there's a lot of value in reports looking consistent from one tournament to another.
